scripts/exportChangelog.gradle

```diff
@@ -4,7 +4,7 @@ task exportChangeLog(
         group: 'docToolchain'
 ) doLast {
     // https://git-scm.com/docs/pretty-formats
-    def changes = "git log --pretty=format:%x7c%x20%ad%x20%n%x7c%x20%an%x20%n%x7c%x20%s%x20%n --date=short ./src/docs/arc42".execute().text
+    def changes = "git log --pretty=format:%x7c%x20%ad%x20%n%x7c%x20%an%x20%n%x7c%x20%s%x20%n --date=short ./resources/reveal.js".execute().text
```

Comment:
hm. I think this should point to `./src/docs`, shouldn't it?
It is the folder for which the changelog should be fetched...

Reply:
Could be, but src/docs/arc42 doesn't exist any more, and other directories can change often. I have chosen reveal.js as external directory, thus having quite stable changelog. Otherwise the tests would need to be adapted after each change.

Reply:
yeah, I will open an issue. We have to move this to the config.
It needs to be the folder for the documentation...